File: Sort Colors.js
var sortColors = function(nums) {
  for(let i = 0 ; i<nums.length ; i++){
    for(let j =0 ; j<nums.length-1 ;j++)
   if(nums[j]>nums[j+1]){
  var temp=nums[j];
  nums[j]=nums[j+1];
  nums[j+1]=temp;
  }
  
  
  }
      return  nums;
    
};

File: Sort Colors.py
class Solution(object):
    def sortColors(self, nums):
        for i in range(len(nums)):
            for j in range(0,len(nums)-1):
                if nums[j] >nums[j+1]:
                    temp=nums[j]
                    nums[j]=nums[j+1]
                    nums[j+1]=temp
        return nums
